src/components/Admin/SystemLogs/SystemLogs.js

```diff
@@ -40,15 +34,26 @@ class SystemLogs extends React.Component {
   }
 
   loadSystemLogs = count => {
-    fetchSystemLogs({ count })
-      .then(payload => {
+    const { accessToken } = this.props;
+    axios
```

Comment:
Why not put this in api folder as we have all the api's there it self

Reply:
It is primarily a problem of axios, I saw an unsolved issue of text type responses as well, so we will have to deal with this in the file itself.

Reply:
Okay